scipy/ndimage/_filters.py

```diff
@@ -433,19 +433,46 @@ def sobel(input, axis=-1, output=None, mode="reflect", cval=0.0):
     %(mode_multiple)s
     %(cval)s
 
+    Notes
+    -----
+    This function computes the axis-specific Sobel gradient.
+    The horizontal edges can emphasised with the horizontal trasform (axis=0),
+    the vertical edges with the vertical transform (axis=1) and so on for higher dimensions.
+    These can be combined to give the magnitude.
+
     Examples
     --------
     >>> from scipy import ndimage, datasets
     >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+
+    >>> ascent = datasets.ascent().astype('int32')
+    >>> sobel_h = ndimage.sobel(ascent, 0)  # horizontal gradient
+    >>> sobel_v = ndimage.sobel(ascent, 1)  # vertical gradient
+    >>> magnitude = np.sqrt(sobel_h**2 + sobel_v**2)
+    >>> magnitude *= 255.0 / np.max(magnitude)  # normalization
+
     >>> fig = plt.figure()
     >>> plt.gray()  # show the filtered result in grayscale
```

Comment:
```suggestion
    >>> fig = plt.figure(figsize=(12, 3))
    >>> plt.gray()  # show the filtered result in grayscale
```

Besides the figure size this looks good to me. Makes it much clearer for the average user.

Reply:
Thanks for the suggestion